### PR TITLE
h1

### PR DESCRIPTION
**Pull request by ahadalmu**
complete the homework: 5/5
comfort: 5/5
+: I learn and practice at the seam time. this make me more comfort with my new knowledge.
-: nothing so far.
notes: nothing thank you.

### CODE REVIEW

File: HW/aboutme.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <div>
+
+        <h3>I ENJOY LIFE AS A DEVELOPER</h3>
+
+        <p>
+            I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in<br>
+            the graphic design world, specializing in the creation of responsive websites.
+        </p>
+
+        <img src="profile pic.png" alt=" profile pic" />
```
Comment (JackieCasper) [inline HW/aboutme.html:28]: try to not use spaces when naming documents or pictures. Instead, rename it "profile_pic.png"

File: HW/aboutme.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About Me</title>
+</head>
+
+<body>
+    <link rel="stylesheet" href="styleaboutme.css">
```
Comment (JackieCasper) [inline HW/aboutme.html:12]: Put your css links in the `<head>` tag, not the `<body>` tag (around line 5-8)

File: HW/aboutme.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    <h2>
+        <a href="Aboutme.html">About Me</a> | <a href="Portfolio.html">Portfollo</a>
```
Comment (JackieCasper) [inline HW/aboutme.html:14]: when naming documents, it is best practice not to use capital letters. "about_me.html" would be better

**Comment by JackieCasper** [comment]
Left a few notes about best practices, but it looks great! 👍

**Comment by ahadalmu** [comment]
Thank you 🙏

**Pull request closed without merging**